framework/src/test/java/org/tron/core/services/filter/LiteFnQueryGrpcInterceptorTest.java

```diff
@@ -89,40 +93,45 @@ public void init() {
   /**
    * destroy the context.
    */
-  @After
-  public void destroy() throws InterruptedException {
+  @AfterClass
+  public static void destroy() throws InterruptedException {
     if (channelFull != null) {
       channelFull.shutdown().awaitTermination(5, TimeUnit.SECONDS);
     }
+    if (channelSolidity != null) {
+      channelSolidity.shutdown().awaitTermination(5, TimeUnit.SECONDS);
```

Comment:
What if the channelSolidity have not been shut down after 5 seconds?

Reply:
Waits for the channel to become terminated, giving up if the timeout is reached.